hbs-helpers.js

```diff
@@ -42,13 +42,27 @@ function localizedBreachDataClasses(supportedLocales, dataClasses, args) {
 
 
 function fluentNestedBold(supportedLocales, id, args) {
-  if (args.hash.breachName) {
-    args.hash.breachName = `<span class="medium">${args.hash.breachName}</span>`;
-  }
+  const saveArgs = JSON.parse(JSON.stringify(args.hash));
+  const stringIds = Object.keys(saveArgs).filter(key => key !== "breachCount");
+
+  const addMarkup = (word) => {
+    return `<span class="medium"> ${word} </span>`;
```

Comment:
Not sure if we need the extra whitespace around the `{$word}`, or if it should be on the outside of the `<span>` tags... Probably moot point, just wanted to make sure we aren't adding extra whitespace anywhere.